pulp_rpm/tests/functional/api/test_download_content.py

```diff
@@ -92,9 +93,11 @@ def test_all(self):
         # …and Pulp.
         client.response_handler = api.safe_handler
 
-        unit_url = cfg.get_hosts('api')[0].roles['api']['scheme']
-        unit_url += '://' + distribution['base_url'] + '/'
-        unit_url = urljoin(unit_url, unit_path)
+        unit_url = reduce(urljoin, (
```

Comment:
Does this change and the one below need to be copied across to the other plugins?

Reply:
@dralley this is not a breaking change, the idea is to gradually change in other plugins to replace those 3 lines with this new method to build the content host URL.

We will send PRs against other plugins to fix that lines soon.